server/src/main/java/org/elasticsearch/client/ClusterAdminClient.java

```diff
@@ -529,7 +529,7 @@
     /**
      * Delete snapshot.
      */
-    DeleteSnapshotRequestBuilder prepareDeleteSnapshot(String repository, String snapshot);
+    DeleteSnapshotRequestBuilder prepareDeleteSnapshot(String repository, String... snapshot);
```

Comment:
`String, String...` is really dirty I know, but it's a massive change-set to adjust this API since we use it in so many places in tests so I figured I'd keep it this way for now.